backend/slackbot/handler.js

```diff
@@ -1,46 +1,47 @@
 'use strict';
-//const hook = 'https://hooks.slack.com/services/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'; // testing slack
 const hook = 'https://hooks.slack.com/services/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'; // group slack
-
 const gatewayURL = process.env.GW_URL;
 const axios = require('axios');
 const queryString = require('query-string');
 
 module.exports = {
   handler,
-  handleCommand,
-  getApps,
-  getStatistics,
-  sendStats,
-  getSentimentOverTime,
 };
 
+/**
+ * Handler function that determines if a http request was invoked by scheduled report
+ * @param event The event or request
+ */
 async function handler (event) {
-  let body = event.body;
-  let responseMessage = '';
+  if (event.hasOwnProperty('detail-type')) {
```

Comment:
Let's see if there's a better a way to check the AWS event type. If not, explain with a comment.